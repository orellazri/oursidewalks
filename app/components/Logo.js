import React from "react";
import { Image, StyleSheet } from "react-native";

export default function Logo() {
  return <Image source={require("../assets/images/logo.png")} style={styles.logo} />;
}

const styles = StyleSheet.create({
  logo: {
    right: 20,
    width: 150,
    resizeMode: "contain",
    alignSelf: "flex-end",
    marginLeft: 40,
    marginTop: 10,
  },
});
