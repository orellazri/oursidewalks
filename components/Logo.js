import React from "react";
import { Image, StyleSheet } from "react-native";

export default function Logo() {
  return <Image source={require("../assets/images/logo.png")} style={styles.logo} />;
}

const styles = StyleSheet.create({
  logo: {
    position: "absolute",
    top: 50,
    right: 20,
    width: 150,
    resizeMode: "contain",
  },
});
