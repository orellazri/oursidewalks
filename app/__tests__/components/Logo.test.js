import renderer from "react-test-renderer";

import Logo from "components/Logo";

describe("<Logo />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
