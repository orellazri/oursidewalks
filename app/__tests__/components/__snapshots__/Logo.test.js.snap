// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "alignSelf": "flex-end",
      "marginLeft": 40,
      "marginTop": 10,
      "resizeMode": "contain",
      "right": 20,
      "width": 150,
    }
  }
/>
`;
